app/lib/actions/katello/repository/update_cv_repo_cert_guard.rb

```diff
@@ -0,0 +1,13 @@
+module Actions
+  module Katello
+    module Repository
+      class UpdateCVRepoCertGuard < Actions::Base
+        include Actions::Katello::PulpSelector
+
+        def plan(repository, smart_proxy)
+          plan_optional_pulp_action([::Actions::Pulp3::Repository::UpdateCVRepositoryCertGuard], repository, smart_proxy)
+        end
+      end
```

Comment:
Can you add a 'humanized_name' method to show a more friendly name in the tasks list?  Maybe something that doesn't reference cert guard, like 'Updating repository authentication configuration' or something...